gtsam/nonlinear/NonlinearFactorGraph.h

```diff
@@ -24,6 +24,7 @@
 #include <gtsam/geometry/Point2.h>
 #include <gtsam/nonlinear/NonlinearFactor.h>
 #include <gtsam/inference/FactorGraph.h>
+#include <gtsam/slam/PriorFactor.h>
```

Comment:
So, this will have to change. Breaks Include DAG. See comments by Jose and me :-)

Reply:
Yep! I changed all

`#include <gtsam/slam/PriorFactor.h>`

to

`#include <gtsam/nonlinear/PriorFactor.h>`